src/app/books/components/book-form/book-form.component.html

```diff
@@ -2,20 +2,20 @@
   <div class="book-create-container">
     <h1 class="book-create__title">Create book</h1>
 
-    <form class="book-create-form" (ngSubmit)="submit()">
+    <form class="book-create-form" [formGroup]="createBookForm" (ngSubmit)="submit()">
       <mat-form-field appearance="legacy" hideRequiredMarker>
```

Comment:
`hideRequiredMarker` is not a kind of suitable solution. Your control doesn't know that it's a required field.